io/eolearn/io/sentinelhub_service.py

```diff
@@ -355,12 +377,46 @@ def __init__(self, sen2cor_classification, layer, **kwargs):
             'SNW': FeatureType.DATA
         }
 
+        if 'image_format' in kwargs:
+            image_format = kwargs.get('image_format')
+            kwargs.pop('image_format', None)
+            sample_type = image_format.get_sample_type()
+        else:
+            image_format = MimeType.TIFF_d8
+            sample_type = 'INT8'
```

Comment:
Instead of `sample_type = 'INT8'` it is better that you do `sample_type = image_format.get_sample_type()` (and move it out of `if` sentence.)
That is because you don't want to hardcode `'INT8'` here since it already exists in sentinelhub-py.

Reply:
Missed this one.